README.md

```diff
@@ -227,45 +202,6 @@ Uncomment and modify the last section in case you're serializing classes with na
 In case you want to exclude serializable classes that are used, but never serialized at runtime,
 you will need to write custom rules with narrower [class specifications](https://www.guardsquare.com/manual/configuration/usage).
 
-<details>
```

Comment:
These rules could be confused. 
Because the rules are already supplied with the library, then it is not entirely clear why you need to add any more duplicate rules.